core/src/main/java/com/linecorp/armeria/server/ServiceConfigSetters.java

```diff
@@ -176,6 +178,14 @@ ServiceConfigSetters blockingTaskExecutor(ScheduledExecutorService blockingTaskE
     @UnstableApi
     ServiceConfigSetters multipartUploadsLocation(Path multipartUploadsLocation);
 
+    /**
+     * Sets the {@link Supplier} which generates a {@link RequestId}.
```

Comment:
```suggestion
     * Sets the {@link Function} which generates a {@link RequestId}.
```

Reply:
I see more than one place that have the same problem. Please fix them all. 🙇

Reply:
I had to fix them after I had pull upstream but I forgot. Thanks for pointing them out!